app/helpers/bgeigie_imports_helper.rb

```diff
@@ -33,4 +33,16 @@ def operation_button(bgeigie_import, action, text = t(format('.%s', action)))
       f.submit text, class: 'btn btn-primary'
     end
   end
+
+  def unmoderated_id_list
```

Comment:
```suggestion
  def unmoderated_id_list(bgeigie_imports)
    bgegie_imports.each_with_object([]) do |import, list|
      list << import.id unless import.approved?
    end
  end
```

It would be nicer not to depend on instace variable `@bgeigie_import`. Instead `unmoderated_id_list` helper should receive list of bGeigie imports like this.

We could use `Array#each_with_object` here.